source/llvm/KineticLawParameterResolver.cpp

```diff
@@ -67,4 +67,12 @@ void KineticLawParameterResolver::recursiveSymbolPop()
     parentResolver.recursiveSymbolPop();
 }
 
+bool KineticLawParameterResolver::isLocalParameter(const std::string& symbol)
+{
+    if (kineticLaw.getLocalParameter(symbol) != NULL) {
+        return true;
+    }
+    return kineticLaw.getParameter(symbol) != NULL;
```

Comment:
Why don't we just return 'false' here?

Reply:
Because there are two different ways that SBML stores local parameters, based on the level and version.  In later level/versions, it's 'getLocalParameter'; in earlier versions it was 'getParameter'.  This checks both.  (For any model with 'rateOf', it would have to be the later 'getLocalParameter'.  But the function name implies *any* local parameters, so I went ahead and have both, in case someone in the future wants it.